hal_ugv/State state
---
# RESULT
bool    success
string  status
